core/server/api_container/server/startosis_engine/kurtosis_instruction/tasks/tasks_shared.go

```diff
@@ -260,3 +260,11 @@ func formatErrorMessage(errorMessage string, errorFromExec string) string {
 	reformattedErrorMessage := strings.Join(splitErrorMessageNewLine, "\n  ")
 	return fmt.Sprintf("%v\n  %v", errorMessage, reformattedErrorMessage)
 }
+
```

Comment:
perhaps we can just do

defer removeService() - and have this log instead of throwing an error.

Reply:
sorry; did not get your comment;; this log is only used at the moment for exec errors.. Are you suggesting instead of throwing error on remove_service, I should just log it @h4ck3rk3y ?

Reply:
you use this function in a defer block upstream and log it; so it never gets "thrown"; so i was wondering if this could just log it. nothing major though.